package.json

```diff
@@ -297,12 +297,13 @@
   },
   "dependencies": {
     "@octokit/rest": "^15.9.5",
-    "node-emoji": "^1.8.1",
+    "axios": "^0.18.0",
```

Comment:
Let's also update the `ThirdPartyNotices.txt` with this

Reply:
I updated it.